weldx/utility.py

```diff
@@ -105,6 +107,32 @@ def _sine(
     return TimeSeries(expr)
 
 
+def lcs_coords_from_ts(
+    ts: TimeSeries, time: Union[pd.DatetimeIndex, pint.Quantity]
+) -> xr.DataArray:  # pragma: no cover
+    """Create translation coordinates from a TimeSeries at specific timesteps.
+
+    Parameters
+    ----------
+    ts:
+        TimeSeries that describes the coordinate motion as a 3D vector.
+    time
+        Timestamps used for interpolation.
+        TODO: add support for pd.DateTimeindex as well
+
+    Returns
+    -------
+    A xarray.DataArray with correctly labeled dimensions to be used for LCS creation.
+
+    """
+    ts_data = ts.interp_time(time=time)
+    # assign vector coordinates and convert to mm
+    ts_data = ts_data.rename({"dim_1": "c"}).assign_coords({"c": ["x", "y", "z"]})
+    ts_data.data = ts_data.data.to("mm").magnitude
+    ts_data["time"] = pd.TimedeltaIndex(ts_data["time"].data)
+    return ts_data
```

Comment:
Maybe I just missed it but shouldn't we keep the unit (`mm`)?

Reply:
I don't think we support units anywhere with regards to LCS (yet)

It's no problem creating a "quantity LCS" here but combining with non unit aware LCS during CSM interpolation throws the expected `DimensionalityError`

Should try to fix this soon, currently not even a default unit for LCS coords is mentioned or agreed upon anywhere